### PR TITLE
Updates to the website formatting

### PR DESCRIPTION
**Pull request by mmurray22**
This PR is focused on the new and improved formatting of the landing page.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -5,63 +5,65 @@
...
+        <div id="basic-profile">
+            <div id="wrapper">
+                <img id="profile" src="/images/PROFILE_PIC.JPG">
+            </div>
+            <h1>Michaela Murray's Portfolio</h1>
+            <div id="intro">
+                <ul style="list-style-type: none;">
+                <li>Stanford Class of 2022</li> 
+                <li>Major: Computer Science</li>
+                <li>STEP Intern @ Google</li>
```
Comment (kaveh096) [inline portfolio/src/main/webapp/index.html]: Indent `li` elements.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -2,49 +2,132 @@
...
   color: white;
-  margin: auto;
-  padding: 1%;
+  margin: 0;
+  min-height: 100%;
+  padding: 0;
+  position: relative;
+}
+
+@keyframes wind{
```
Comment (kaveh096) [inline portfolio/src/main/webapp/style.css]: nit: space before bracket?

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -2,49 +2,132 @@
...
+  align-items: center;
+  background-color: rgba(128, 128, 128, .6);
+  display: inline-flex;
+  justify-content: center;
+  min-height: 36px;
+  text-align: center;
+  
+}
+
+
```
Comment (kaveh096) [inline portfolio/src/main/webapp/style.css]: nit: extra newline

**Review by kaveh096 — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/style.css]: Changed! thanks!

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/style.css]: Changed! Thanks!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/index.html]: thanks! Changed!

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -5,63 +5,65 @@
     <!-- Imported Libraries: Google's Material Design Library and Google's Fonts Library -->
     <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```
Comment (ccondit) [inline portfolio/src/main/webapp/index.html]: How is jquery used in this PR?

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -2,49 +2,131 @@
 @include button.core-styles;
 @use "@material/card";
 @include card.core-styles;
+@import url('https://fonts.googleapis.com/css?family=Lato:100,100i,300,300i,400,400i,700,700i,900,900i');
 
+html {
+   background-size: cover;
+   background-color: #6495ED;
```
Comment (ccondit) [inline portfolio/src/main/webapp/style.css]:
```suggestion
   background-color: #6495ED;
   background-size: cover;
```

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -2,49 +2,131 @@
...
+   height:100%;
+}
+
+/*
+ * Credit to codepen: https://codepen.io/valentin/pen/fkdCG
+*/
```
Comment (ccondit) [inline portfolio/src/main/webapp/style.css]:
```suggestion
 */
```

File: portfolio/src/main/webapp/style.css [hunk 7]
```diff
@@ -2,49 +2,131 @@
...
 body {
-  background-color: #8EC5FC;
-  background-image: linear-gradient(62deg, #8EC5FC 0%, #E0C3FC 100%);
+  animation: wind 20s linear infinite;
+  background-color: #808080;
+  background: url(https://static.radulescu.me/examples/clouds/clouds1000.png);
+  background: url(https://static.radulescu.me/examples/clouds/clouds1000.png) 0 200px,
```
Comment (ccondit) [inline portfolio/src/main/webapp/style.css]: can these two background property be consolidated into one?

File: portfolio/src/main/webapp/style.css [hunk 8]
```diff
@@ -2,49 +2,131 @@
...
+}
+
+/*
+ * Credit to codepen: https://codepen.io/valentin/pen/fkdCG
+*/
 body {
-  background-color: #8EC5FC;
-  background-image: linear-gradient(62deg, #8EC5FC 0%, #E0C3FC 100%);
+  animation: wind 20s linear infinite;
+  background-color: #808080;
```
Comment (ccondit) [inline portfolio/src/main/webapp/style.css]: nit - `background-color` should be alphabetized after `background`

File: portfolio/src/main/webapp/style.css [hunk 9]
```diff
@@ -2,49 +2,131 @@
...
   margin: auto;
   width: 300px;
 }
 
-img {
+#info {
+   border: 1px solid black;
+   margin-top: -50px;
+   margin-right: auto;
+   margin-left: auto;
```
Comment (ccondit) [inline portfolio/src/main/webapp/style.css]: nit - alphabetize these properties

File: portfolio/src/main/webapp/style.css [hunk 10]
```diff
@@ -2,49 +2,131 @@
...
   border: 1px solid white;
   box-shadow: 0 19px 30px #0000004C, 0 15px 12px #00000038;
   height: 400px;
+  margin: 10px;
+  position: relative;
   width: 350px;
+  transform: rotateX(0) rotateY(0);
+  transform-style: preserve-3d;
+  transition-duration: 0.1s;
+  transition-timing-function: ease !important;
```
Comment (ccondit) [inline portfolio/src/main/webapp/style.css]: Is this for the parallax? If so, remove it from this cl about formatting

**Review by ccondit — CHANGES_REQUESTED** [review]
Might still be seeing some mixed commits, but generally seems better!

Also note that no PR is too small. Adding the cloud background would be a great single PR

Updating the formatting would be another one.

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/index.html]: That is an excellent point-it is not 🤦 I will remove that-thanks for pointing it out!

File: portfolio/src/main/webapp/style.css (hunk 8 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/style.css]: thanks! updated!

File: portfolio/src/main/webapp/style.css (hunk 9 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/style.css]: thanks! updated!

File: portfolio/src/main/webapp/style.css (hunk 10 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/style.css]: ah ok I do believe it is-I will remove it. Thanks!

File: portfolio/src/main/webapp/style.css (hunk 7 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/style.css]: Yes indeed! thanks & updated!

**Review by kaveh096 — COMMENTED** [review]
Is this PR still needs a review? If no can you please close it? On my dashboard it shows up as "Needs a Review" but it also still has open comments. If this PR still needs a review, can you please address the existing open comments and then mark those comments as resolved? Thank you.

**Comment by kaveh096** [comment]
Actually, ignore my request to close resolved comments. Responding to them is enough. I'll go ahead and close the once that are addressed.

**Comment by mmurray22** [comment]
Thanks for notification Kaveh! That works for me in the future. Yes, this PR still needs to be approved in order to merge with the master!

File: portfolio/src/main/webapp/index.html [hunk 11]
```diff
@@ -12,56 +12,57 @@
...
-      </div>
-      <div id="intro-button-two" style="display:inline-block;">
-	<div class="mdc-touch-target-wrapper" style="display:block;">
-	   <button class="mdc-button mdc-button--outlined" onclick="blog()" style="color:white;">  
-	      <span class="mdc-button__ripple">Blog</span>
-	   </button>
+        <div id="profile-wrapper" style="display:inline-block;">
+        <div id="basic-profile">
+            <div id="wrapper">
+                <img id="profile" src="/images/PROFILE_PIC.JPG">
```
Comment (kaveh096) [inline portfolio/src/main/webapp/index.html]: Add an `alt` for image [as a fallback](https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback).

File: portfolio/src/main/webapp/index.html [hunk 12]
```diff
@@ -12,56 +12,57 @@
...
+            </div>
+            <h1>Michaela Murray's Portfolio</h1>
+            <div id="intro">
+                <ul style="list-style-type: none;">
```
Comment (kaveh096) [inline portfolio/src/main/webapp/index.html]: Consider moving styles to css per the [separation of concern](https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns) guide.

**Review by kaveh096 — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html (hunk 11 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/index.html]: thanks! updated!

File: portfolio/src/main/webapp/index.html (hunk 12 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/index.html]: thanks! updated!

File: portfolio/src/main/webapp/index.html [hunk 13]
```diff
@@ -12,7 +12,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getComments();createMap();">
```
Comment (ccondit) [inline portfolio/src/main/webapp/index.html:15]: is createmap defined? or is that from another branch?

File: portfolio/src/main/webapp/index.html [hunk 14]
```diff
@@ -13,68 +13,77 @@
...
+            </div>
+            <div class="mdc-touch-target-wrapper">
+                <button id="picture-button" class="mdc-button mdc-button--outlined">  
+                    <span class="mdc-button__ripple">Pictures</span>
+                </button>
+            </div>
+	          <div id="icons">
+              <ul> 
+                <li> 
+                    <a href="https://github.com/mmurray22" target="_blank" >
```
Comment (ccondit) [inline portfolio/src/main/webapp/index.html]: nit - some of these elements have a space between the last attribute and the end of the element

**Review by ccondit — APPROVED** [review]
I'm fine to merge this with a note that I think smaller, focused PRs will be better moving forward. No PR is to small - one to update the structure and then one to update the style, for instance...

File: portfolio/src/main/webapp/index.html (hunk 13 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/index.html:15]: another branch that is merged into master!

File: portfolio/src/main/webapp/index.html (hunk 14 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/index.html]: ah ok noted! will address :)

**Review by kaveh096 — APPROVED** [review]

**Pull request merged**